tests/console/validate_lvm.pm

```diff
@@ -29,8 +30,8 @@ sub run {
     assert_script_run('lvmconfig --mergedconfig --validate | grep "LVM configuration valid."',
         fail_message => 'LVM config validation failed');
 
-    record_info('LVM volume', 'Verify the LVM physical volume exists');
-    assert_script_run('lvmdiskscan -v | grep "1 LVM physical volume"',
+    record_info('LVM volume', 'Verify the LVM physical volume(s) exist');
+    assert_script_run('lvmdiskscan -v | egrep "[[:digit:]] LVM physical volume[s]{0,1}$"',
```

Comment:
we should fix expectation, otherwise, when it will find a different number we will have false positive for the output of this command. Besides that I'd recommend to pluralize fail_message.

Reply:
By fixed expectations, do you mean test data ? As otherwise, I'd keep it like this, because precisely the intention is to make this work even if we have one or more VGs. Then to verify that we have what is expected is covered by verify_lvm_partitions.pm.

Reply:
So for regexp, I would say that we can leave just `LVM physical volume` as we accept both, otherwise I would suggest to simplify pluralization part with just `s?` so it's `volumes?$`, `?` denotes zero or one occurrence, so it's exactly same as `{0,1}`.